src/interfaces/game.interface.ts

```diff
@@ -20,7 +18,7 @@ export interface IGameCreateSchema {
 }
 
 export interface GameOptions {
-  maxPlayersPerTeam: any;
+  maxPlayersPerTeam: number;
```

Comment:
Why do you change already existing game schema? Or why dont you change README.md and schemas folder?

Reply:
I am about field that is not in schemas folder or was not documented in README.md. We dont have this field `maxPlayersPerTeam`, so either avoid it or document this field